### PR TITLE
Replaces summary report with GitHub workflow checks

### PR DESCRIPTION
**Pull request by ebenoist**
## What
Align PR and issue workflow labels to allow audits to be performed in GitHub

* Secure console access will now require `emergency-approval`
* Secure console audits will check for that tag and the application of `posthoc-approval`

Related: https://github.com/reverbdotcom/reverb/pull/26763

### CODE REVIEW

File: src/check_for_review.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        return;
+      }
+    }
+
+    await addCommentToIssue(
+      issue.number,
+      msg,
+    );
+
+    await labelIssue(issue.number, missingApprovalLabel);
```
Comment (stevenweber) [inline src/check_for_review.ts:33]: Why do we need to apply a label that it's missing a review -- is the idea that the post-hoc reviewer adds the posthoc review label and then removes the missing approval label?

File: src/check_for_review.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+export async function checkForReview() {
+  const msg = `This issue is missing verification by a peer! Have a peer review this issue and apply the ${posthocApprovalLabel} to approve.`;
+  const issues = await getIssuesMissingReview();
+
+  return Promise.all(issues.map(async (issue) => {
+    if (issue.pull_request) {
+      const detail = await getDetailedPR(issue.number);
+      if (!detail.merged) {
+        return;
```
Comment (stevenweber) [inline src/check_for_review.ts:24]: Is the idea to followup with workflow for issues, i.e. write console requests?

File: src/check_for_review.ts (hunk 1 above)
Comment (ebenoist) [reply src/check_for_review.ts:33]: The label helps identify those PRs that don't have a post-hoc review, which makes the auditing task here simple (e.g. filter to prs with this tag and not the approval tag)

File: src/check_for_review.ts (hunk 2 above)
Comment (ebenoist) [reply src/check_for_review.ts:24]: Not sure what that means, let's follow up?

Comment (stevenweber) [reply src/check_for_review.ts:24]: Misread this conditional.

Comment (ebenoist) [reply src/check_for_review.ts:22]: No worries! I think this should be gtg now

**Pull request merged**